daemon/container_unix.go

```diff
@@ -896,8 +900,6 @@ func (container *Container) configureNetwork(networkName, service, networkDriver
 		}
 	}
 
-	container.updateSandboxNetworkSettings(sb)
```

Comment:
@aboch good question. I think instead of removing this call, we should move this under a check 
` if !execdriver.SupportsHooks()`. And for native driver, we should set the sandbox-id here and set the sandbox-key `(container *Container) setExternalKey`. WDYT ?